contents/handbook/growth/marketing/in-app.md

```diff
@@ -0,0 +1,41 @@
+---
+title: In-App
+sidebar: Handbook
+showTitle: true
+---
+
+Occasionally, we use in-app messages to tell users about certain things. We recognize that in-app messages can be intrusive and we want to avoid spamming our users with too many of them, too frequently. For that reason, we're judicious about the way in which we use them. 
+
+We currently don't have a separate system for tracking in-app messages, so Marketing currently owns the channel and is responsible for ensuring that messages aren't used excessively.
+
+## Types of in-app message
+
+Currently, there are three ways in which we can send in-app messages.
```

Comment:
There's actually a 4th and it's the user interview popup thing.

Reply:
Yeah, I think we're going to purposefully exclude that from here as it's used for very different purposes.